src/modules/rover_differential/module.yaml

```diff
@@ -162,7 +162,12 @@ parameters:
 
       RD_TRANS_DRV_TRN:
         description:
-            short: Heading error threshhold to switch from driving to spot turning
+            short: Yaw error threshhold to switch from driving to spot turning
```

Comment:
I don't like that we have another threshold with a hard-switch in behavior, but don't have a good idea how we can improve it for waypoint following (path following: limit speed based on curvature of path ahead).

Reply:
The speed is limited based on the curvature of the path in the ackermann module.
With differential it is on purpose that it comes to a fullstop to fully utilize the ability of a differential rover to turn on the spot (this reduces the accumulated crosstrack error because the rover doesn't need to cut corners like an ackermann rover).
Once we start to think about offering different guidance logics we could add a version that makes differential rovers cut corners the same way ackermann does if travel time is a higher priority than crosstrack error minimization.

Reply:
yes sounds very reasonable, you seem to be on top of the it.